ros-speech_recognition/install.bash

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+url=$(curl -s https://api.github.com/repos/tue-robotics/speech_recognition/releases/latest | \
```

Comment:
Check ros-dashboard target, it uses some python to do this.

Reply:
It is pretty much the same. Here I am specifying the version to always be the latest one while there it takes a version as an argument and searches over all the versions.

What do you want me to do? That's an overkill for what I am doing here.

Reply:
True, but it uses json in python is cleaner IMO.